daemons/lvmdbusd/lv.py

```diff
@@ -304,16 +322,14 @@ def VolumeType(self):
 					'V': 'thin Volume', 't': 'thin pool', 'T': 'Thin pool data',
 					'e': 'raid or pool metadata or pool metadata spare',
 					'-': 'Unspecified'}
-		return dbus.Struct((self.state.Attr[0], type_map[self.state.Attr[0]]),
-						signature="as")
```

Comment:
Tony, I am not yet familiar with all this, but the signature changes from `as` to `(ss)` IIUC. Is that correct?

Reply:
@csonto Correct this signature changed as the existing is signature is actually a bug.  I checked with `libblockdev` and they are not using this property, otherwise we would have likely caught the bug.